universum.cabal

```diff
@@ -1,5 +1,5 @@
 name:                universum
-version:             1.2.0
+version:             0.1.0
 synopsis:            Custom prelude
```

Comment:
You also need to add CHANGELOG to extra-doc-files here. And also change GHC-8.4.2 to GHC-8.4.3 in `.cabal` file